stl/inc/xatomic.h

```diff
@@ -50,6 +50,20 @@ _STL_DISABLE_CLANG_WARNINGS
 #define _MT_INCR(x) _INTRIN_RELAXED(_InterlockedIncrement)(reinterpret_cast<volatile long*>(&x))
 #define _MT_DECR(x) _INTRIN_ACQ_REL(_InterlockedDecrement)(reinterpret_cast<volatile long*>(&x))
 
+// The following macros are SHARED with vcruntime and any updates should be mirrored.
+// Also: if any macros are added they should be #undefed in vcruntime and <atomic> as well.

```

Comment:
I forgot that `<xmemory>` also includes `<xatomic.h>`. Probably more investigation is needed.

Reply:
`#undef` directives are dropped.